pkgs/development/python-modules/libusb1/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, lib, buildPythonPackage, fetchPypi, libusb1 }:
+
+buildPythonPackage rec {
+  name = "${pname}-${version}";
+  version = "1.6.4";
+  pname = "libusb1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "03b7xrz8vqg8w0za5r503jhcmbd1ls5610jcja1rqz833nf0v4wc";
+  };
+
+  patchPhase = lib.optionalString stdenv.isLinux ''
+    substituteInPlace usb1/libusb1.py --replace \
+      "ctypes.util.find_library(base_name)" "'${libusb1}/lib/libusb-1.0.so'"
```

Comment:
see `${stdenv.hostPlatform.extensions.sharedLibrary}`

Reply:
That was fast! Thank you. I don't know how to use this. Any examples?

Reply:
`python-packages.nix` has some examples.

Reply:
Ok, done.